frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -115,24 +142,27 @@ export const FormComponentConfig = ({
           handleComponentChange={handleComponentUpdate}
         />
       )}
+      {booleanRestPropertyKeys.map((propertyKey) => {
+        if (unsupportedPropertyKeys.includes(propertyKey) || !rest[propertyKey]) return null;
+        return (
+          <EditBooleanValue
+            component={component}
+            handleComponentChange={handleComponentUpdate}
+            propertyKey={propertyKey}
+            key={propertyKey}
+            helpText={rest[propertyKey]?.description}
+          />
+        );
+      })}
 
-      {Object.keys(rest).map((propertyKey) => {
-        if (!rest[propertyKey]) return null;
+      {/** String and number fields (incl. arrays with enum values) */}
+      {remainingRestPropertyKeys.map((propertyKey) => {
+        if (unsupportedPropertyKeys.includes(propertyKey) || !rest[propertyKey]) return null;
         if (
-          rest[propertyKey].type === 'boolean' ||
-          rest[propertyKey].$ref?.endsWith(ExpressionSchemaBooleanDefinitionReference)
+          rest[propertyKey].type === 'number' ||
+          rest[propertyKey].type === 'integer' ||
+          rest[propertyKey].$ref?.endsWith(getExpressionSchemaDefinitionReference('number'))
```

Comment:
Would it make sense to filter out the properties that are numberValue with the same `getFilteredProperties` method and render them in its own section?